components/ShakeApi.js

```diff
@@ -0,0 +1,101 @@
+import React, { useState, useEffect } from "react";
+import { View, Text, ActivityIndicator } from "react-native";
+import styled from "styled-components/native";
+import { Accelerometer } from "expo-sensors";
+
+const Container = styled.View`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: pink;
+  padding: 10px;
+  width: 100%;
+  height: 100%;
+  text-align: center;
+`;
+
+const BoredText = styled.Text`
+  font-weight: 800;
+  text-align: center;
+  text-transform: uppercase;
+  margin-bottom: 20;
+`;
+
+const ShakeText = styled.Text`
+  font-weight: 700;
+  text-align: center;
+  margin-bottom: 10;
+`;
+
+const ShakeApi = () => {
+  const [data, setData] = useState({
+    x: 0,
+    y: 0,
+    z: 0,
+  });
+
+  const [bored, setBored] = useState({});
+  const [loading, setLoading] = useState(false);
+  const [subscription, setSubscription] = useState(null);
+
+  useEffect(() => {
```

Comment:
It is a lot of code from here to the return statement, maybe it would be nice to have a comment directly over each function that describes what is happening? 

I'm usually not commenting on my own code, but am now thinking that I will start, for my own sake as well =)